src/Monaco/mylang.js

```diff
@@ -0,0 +1,579 @@
+  const mylangKeywords = [
+    {
+      label: 'continue',
+      kind: monaco.languages.CompletionItemKind.Keyword,
+      insertText: 'continue'
+    },
+    {
+      label: 'for',
+      kind: monaco.languages.CompletionItemKind.Keyword,
+      insertText: 'for',
+    },
+    {
+      label: 'and',
+      kind: monaco.languages.CompletionItemKind.Keyword,
+      insertText: 'and',
+    },
+    {
+      label: 'or',
+      kind: monaco.languages.CompletionItemKind.Keyword,
+      insertText: 'or',
+    },
+    {
+      label: 'if',
+      kind: monaco.languages.CompletionItemKind.Keyword,
+      insertText: 'if',
+    },
+    {
+      label: 'break',
+      kind: monaco.languages.CompletionItemKind.Keyword,
+      insertText: 'break',
+    },
+    {
+      label: 'else',
+      kind: monaco.languages.CompletionItemKind.Keyword,
+      insertText: 'else',
+    },
+    {
+      label: 'return',
+      kind: monaco.languages.CompletionItemKind.Keyword,
+      insertText: 'return',
+    },
+    {
+      label: 'const',
+      kind: monaco.languages.CompletionItemKind.Keyword,
+      insertText: 'const',
+    },
+    {
+      label: 'while',
+      kind: monaco.languages.CompletionItemKind.Keyword,
+      insertText: 'while',
+    },
+    {
+      label: 'true',
+      kind: monaco.languages.CompletionItemKind.Keyword,
+      insertText: 'true',
+    },
+    {
+      label: 'false',
+      kind: monaco.languages.CompletionItemKind.Keyword,
+      insertText: 'false',
+    },
+  ];
+
+  /** @todo Are these really necessary / needed in medcoedelogic? */
+  // Define the list of type keywords
+  const mylangTypeKeywords = [
+    {
+      label: "boolean",
+      kind: monaco.languages.CompletionItemKind.TypeParameter,
+      insertText: "boolean"
+    },
+    {
+      label: "double",
+      kind: monaco.languages.CompletionItemKind.TypeParameter,
+      insertText: "double"
+    },
+    {
+      label: "byte",
+      kind: monaco.languages.CompletionItemKind.TypeParameter,
+      insertText: "byte"
+    },
+    {
+      label: "int",
+      kind: monaco.languages.CompletionItemKind.TypeParameter,
+      insertText: "int"
+    },
+    {
+      label: "short",
+      kind: monaco.languages.CompletionItemKind.TypeParameter,
+      insertText: "short"
+    },
+    {
+      label: "char",
+      kind: monaco.languages.CompletionItemKind.TypeParameter,
+      insertText: "char"
+    },
+    {
+      label: "void",
+      kind: monaco.languages.CompletionItemKind.TypeParameter,
+      insertText: "void"
+    },
+    {
+      label: "long",
+      kind: monaco.languages.CompletionItemKind.TypeParameter,
+      insertText: "long"
+    },
+    {
+      label: "float",
+      kind: monaco.languages.CompletionItemKind.TypeParameter,
+      insertText: "float"
+    },
+  ];
+
+  // Define the list of functions
+  const mylangFunctions = [
+    { 
+      label: 'inList', 
+      kind: monaco.languages.CompletionItemKind.Function,
+      insertText: 'inList()', 

```

Comment:
label and insertText should be as provided by us, i.e. "in list" respectively "in list()"